src/server/tiered_storage.cc

```diff
@@ -475,18 +427,10 @@ void TieredStorage::Delete(DbIndex dbid, PrimeValue* value) {
   DCHECK(value->IsExternal());
   ++stats_.total_deletes;
 
-  tiering::DiskSegment segment = value->GetExternalSlice();
```

Comment:
why this was removed?

Reply:
Indeed, it can be triggered via a non command